_posts/fr/newsletters/2023-07-26-newsletter.md

```diff
@@ -0,0 +1,144 @@
+  discussions de la récente réunion des développeurs LN à New York. Certaines des discussions ont porté sur les sujets suivants :
+  {% assign timestamp="10:48" %}
+
+    - *Confirmation fiable des transactions :* [relais de paquets][topic package relay], [relais de transactions
+      v3][topic v3 transaction relay], [points d'ancrage éphémères][topic ephemeral anchors], [mempool en
+      grappe][topic cluster mempool] et d'autres sujets liés au relais de transactions et à l'extraction minière ont été discutés
+      dans le contexte de la manière dont ils permettront de confirmer de manière plus fiable les transactions LN onchain, sans la
+      menace d'épinglage de [transaction][topic transaction pinning] ou la nécessité de payer des frais excessifs lors de
+      l'utilisation de [CPFP][topic cpfp] ou de [RBF][topic rbf] pour augmenter les frais. Nous recommandons vivement aux lecteurs
+      intéressés par la politique de relais de transactions, qui affecte presque tous les protocoles de deuxième couche, de lire
+      les notes pour les commentaires éclairés fournis par les développeurs LN sur plusieurs initiatives en cours.
+
+    - *Canaux Taproot et MuSig2 :* une brève discussion sur l'avancement des canaux utilisant des sorties [P2TR][topic taproot]
+      et des signatures [MuSig2][topic musig]. Une partie importante des notes de cette discussion portait sur un protocole de
+      fermeture mutuelle simplifié ; voir l'article précédent pour l'un des résultats de cette discussion.
+
+    - *Annonces de canaux mises à jour :* le protocole de diffusion LN ne relaie actuellement les annonces de nouveaux canaux ou
+      de canaux mis à jour que si ces canaux ont été financés à l'aide d'une sortie P2WSH qui s'est engagée à un script
+      `OP_CHECKMULTISIG` 2-sur-2. Pour passer aux sorties [P2TR][topic taproot] avec des engagements de [multisignature sans
+      script][topic multisignature] basés sur [MuSig2][topic musig], le protocole de diffusion devra être mis à jour. Un
+      [sujet][topic channel announcements] a également été discuté lors de la précédente réunion en personne des développeurs LN
+      (voir [Newsletter #204][news204 gossip]) pour savoir s'il convient de faire une mise à jour minimale du protocole (appelée
+      v1.5 gossip) qui ajoute simplement la prise en charge des sorties P2TR, ou une mise à jour plus générale du protocole
+      (appelée v2.0) qui permet plus largement l'utilisation d'une signature valide pour n'importe quelle UTXO de n'importe quel
+      type pour les annonces. Permettre l'utilisation de n'importe quelle sortie signifie que la sortie utilisée pour annoncer
+      le canal est moins susceptible qu'aujourd'hui d'être la sortie réellement utilisée pour exploiter le canal, rompant ainsi
+      le lien public entre les sorties et le financement du canal.
+
+Une autre considération discutée était de savoir si une UTXO d'une valeur de _n_ devrait être autorisée à annoncer un canal
```

Comment:
@Copinmalin the markdown formatting for this bullet and the ones that follow is off. Can you correct to match the formatting for the above few bullets?